### DISCUSSION TITLE
[Enhancement] Local Playback of Music

### DISCUSSION THREAD

**Discussion by coredev-uk** [ciderapp/Cider#680] [Feature Request]
I see, thanks for clarifying. I posted on the Last.fm subreddit today about the internal scrobbling feature of Apple Music Electron, that's why I wanted to be sure:
https://www.reddit.com/r/lastfm/comments/rj1a42/apple_music_electron_with_internal_scrobbling/
It would be great if your app could also play local music files and scrobble them, because Last.fm users often report problems when using iTunes on Windows or macOS.

__Originally posted by @hans-juergen in https://github.com/ciderapp/Apple-Music-Electron/issues/223#issuecomment-997277494__

**Comment by quacksire** [comment]
https://stackoverflow.com/questions/57518133/play-local-audio-files-in-my-app-without-using-the-input-type-file


https://www.npmjs.com/package/@sequencemedia/itunes-library-stream
